package.json

```diff
@@ -106,15 +106,15 @@
     "grunt-todo": "0.5.0",
     "htmlparser2": "3.9.0",
     "husky": "0.11.4",
-    "intern": "3.4.2",
+    "intern": "3.4.3",
     "npmshrink": "1.0.1",
     "proxyquire": "1.7.4",
     "sinon": "1.17.3",
     "sync-exec": "0.6.2",
     "xmlhttprequest": "git://github.com/zaach/node-XMLHttpRequest.git#onerror"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=4.8.0"
```

Comment:
he.. How'd we never do this before? Will this cause any blowups on any outstanding fxa-dev boxes that don't yet have node 4.8?